packages/inferno-animation/src/utils.ts

```diff
@@ -0,0 +1,206 @@
+declare global {
+  // Setting `window.__DEBUG_ANIMATIONS__ = true;` disables animation timeouts
+  // allowing breakpoints in animations for debugging.
+  // https://mariusschulz.com/blog/declaring-global-variables-in-typescript
+  var __DEBUG_ANIMATIONS__: Boolean;
+}
+
+export function addClassName (node: HTMLElement, className: string) {
+  if (className) {
+    const tmp = className.split(' ')
+    for (let i=0; i < tmp.length; i++) {
+      node.classList.add(tmp[i])
+    }
+  }
+}
+
+export function removeClassName (node: HTMLElement, className: string) {
+  if (className) {
```

Comment:
it should probably check isString(className) -- to make 0 pass?

Reply:
Will investigate